ui/raidboss/data/04-sb/dungeon/hells_lid.js

```diff
@@ -83,7 +107,184 @@
+        'Whipping Whittret': 'Wieselpeitsche',
+        'Whisper In The Wind': 'Flüstern des Windes',
+      },
+    },
+    {
+      'locale': 'fr',
+      'replaceSync': {
+        'Demonsgate': 'Porte des Oni',
+        'The Furnace': 'Chaudron des enfers',
+        'Genbu': 'Genbu',
+        'Kamaitachi': 'Kamaitachi',
+        'Otake-Maru': 'Ôtake Maru',
+        'The Polished Shell': 'Vénérable Carapace',
+        'Volcanic Debris': 'Débris volcanique',
+      },
+      'replaceText': {
+        '10-Tonze Slash': 'Taillade de 10 tonz',
+        '100-Tonze Swing': 'Swing de 100 tonz',
+        'Caduceus': 'Caducée',
+        'Circling Winds': 'Vents circulaires',
+        'Disrobe': 'Déshabillage',
+        'Divine Cataract': 'Cataracte divine',
+        'Eruptive Leap': 'Bond éruptif',
+        'Hell Of Waste': 'Enfer de vent',
+        'Hell Of Water': 'Enfer d\'eau',
+        'Hell Of Waves': 'Enfer de vagues',
+        'Late Harvest': 'Moisson tardive',
+        'Reaper\'s Gale': 'Rafale fauchante',
+        'Rolling Winds': 'Vents perçants',
+        'Shell Shower': 'Pluie de carapaces',
+        'Sinister Tide': 'Marée sinistre',
+        'Stone Cudgel': 'Gourdin de pierre',
+        'The Patient Blade': 'Lame patiente',
+        'Volcanic Debris': 'Débris volcanique',
+        'Whipping Whittret': 'Fouine fouettante',
+        'Whisper In The Wind': 'Murmure du vent',
+      },
+    },
+    {
+      'locale': 'ja',
+      'replaceSync': {
+        'Demonsgate': '鬼之門',
+        'The Furnace': '獄之釜',
+        'Genbu': '玄武',
+        'Kamaitachi': 'カマイタチ',
+        'Otake-Maru': 'オオタケ丸',
+        'The Polished Shell': '亀甲舞台',
+        'Volcanic Debris': '噴石',
+      },
+      'replaceText': {
+        '10-Tonze Slash': '10トンズ・スラッシュ',
+        '100-Tonze Swing': '100トンズ・スイング',
+        'Caduceus': '蛇杖撃',
+        'Circling Winds': '疾風衝',
+        'Disrobe': '無手法',
+        'Divine Cataract': '玄天武水壁',
+        'Eruptive Leap': '噴火飛び',
+        'Hell Of Waste': '玄冥瀑',
+        'Hell Of Water': '玄冥流',
+        'Hell Of Waves': '玄冥波',
+        'Late Harvest': '野鎌',
+        'Reaper\'s Gale': '鎌風',
+        'Rolling Winds': '突風衝',
+        'Shell Shower': '亀甲流',
+        'Sinister Tide': '迷潮流',
+        'Stone Cudgel': '岩棍棒',
+        'The Patient Blade': '構え太刀',
+        'Volcanic Debris': '噴石',
+        'Whipping Whittret': '鞭イタチ',
+        'Whisper In The Wind': '風の呼び声',
+      },
+    },
+    {
+      'locale': 'cn',
+      'replaceSync': {
+        'Demonsgate': '鬼之门',
+        'The Furnace': '狱之釜',
+        'Genbu': '玄武',
+        'Kamaitachi': '镰鼬',
+        'Otake-Maru': '大岳丸',
+        'The Polished Shell': '龟甲舞台',
+        'Volcanic Debris': '喷石',
+      },
+      'replaceText': {
+        '10-Tonze Slash': '十吨挥打',
+        '100-Tonze Swing': '百吨回转',
+        'Caduceus': '蛇杖击',
+        'Circling Winds': '疾风冲',
+        'Disrobe': '徒手',
+        'Divine Cataract': '玄天武水壁',
+        'Eruptive Leap': '噴火飛び',
```

Comment:
```suggestion
        'Eruptive Leap': '爆发跳跃',
```

Reply:
Because this skill has no damage and will not be displayed, the Japanese translation word is reserved in the CN client, but I think it is better to translate it into Chinese by ourselves.